fontforgeexe/freetypeui.h

```diff
@@ -0,0 +1,74 @@
+#ifndef FONTFORGE_FONTFORGEEXE_FREETYPEUI_H
+#define FONTFORGE_FONTFORGEEXE_FREETYPEUI_H
+
+#include <fontforge-config.h>
```

Comment:
Hmm. No idea how that happened. @rrthomas?

If other files already do it that way and it passes Travis, there's no need to worry about it for now.